### DISCUSSION TITLE
Relationship PostgreSQL

### DISCUSSION THREAD

**Discussion by JeroenvdBurg** [Budibase/budibase#6545] [Help] [answered]
Hi,

We use the PostgreSQLconnector and have two tables (Table A and Table B) where Table A contains records which are related to Table B. 

So many rows in Table A<> one row in Table B. 

We can create this relationship perfectly in budibase. However because Table A contains a lot of records (3000+) we don't want to have a relationship column in Table B because loading the overview of Table B (with only 10 records)  loads all the related rows from table A , this takes a lot of time. 

When we delete the column in Table B the relationship column in Table A is also removed. Is it possibile to only have a relationship column in Table A ?

**Comment by melohagan** [comment, marked as answer]
Hi @JeroenvdBurg 

So a couple of ideas.

**Easy way**: Just use a custom query (https://docs.budibase.com/docs/custom-queries) to load the overview of Table B.
E.g. `SELECT * FROM TableB;`
No need for joins here!

**Hard way**. Normalise the tables (with an intermediary middle table): https://github.com/Budibase/budibase/discussions/5299

**Comment by JeroenvdBurg** [comment]
Thnx for the quick reply! 

for the custom query: we loose the build in paging functionality 😢 

for the hard-way: creative solution but this feels quite an overkill for a simple many-to-one relation. 

As a feature requests it would be great if we could define the relational binding when creating the relationship.
- two way binding (data is retreived in both Tables)
- one way Table A (relational data (from Table B) is only retreived and shown in Table A)
- one way Table B (relational data (from Table A) is only retreived and shown in Table B)

**Reply by melohagan** [reply]
That's a good idea! Would you mind raising that as a feature request here please? https://github.com/Budibase/budibase/issues/new/choose
